CNAME

```diff
@@ -1 +0,0 @@
-www.4334.ca
```

Comment:
This needs to be here for 4334.ca to work

Reply:
It's www.4334.ca on the current site

Reply:
Sorry I read it wrong